client/src/components/layers/EditAnnotationLayer.ts

```diff
@@ -150,6 +155,7 @@ export default class EditAnnotationLayer extends BaseLayer<GeoJSON.Feature> {
   formatData(frameData: FrameDataTrack[]) {
     this.selectedHandleIndex = -1;
     this.hoverHandleIndex = -1;
+    this.$emit('update:selectedIndex', this.selectedHandleIndex);
```

Comment:
ditto

Reply:
same response as above, it is syncing the "-1" in the `usemodeManager` where it wasn't before.